src/NPacMan.Game/Game.cs

```diff
@@ -60,10 +56,10 @@ public static Game Create()
         public PacMan PacMan { get; private set; }
 
         public IReadOnlyCollection<CellLocation> Coins
-            => _settings.Coins.Except(_collectedCoins).ToList().AsReadOnly();
```

Comment:
This code was being hit several times per second,  and each time created a new list!